go.mod

```diff
@@ -72,4 +72,5 @@ require (
 	google.golang.org/protobuf v1.28.0
 	gopkg.in/inf.v0 v0.9.1
 	gopkg.in/yaml.v2 v2.4.0
+	k8s.io/release v0.13.0
```

Comment:
I know this is not for testing but could we vendor this under `test/tools/`, adding this to the main module will force this into all podman module users (pkg/bindings for example) or disto packagers like debian.